Sources/NIO/EventLoop.swift

```diff
@@ -633,7 +633,8 @@ enum NIORegistration: Registration {
 }
 
 /// Execute the given closure and ensure we release all auto pools if needed.
-private func withAutoReleasePool<T>(_ execute: () throws -> T) rethrows -> T {
+@inlinable
+internal func withAutoReleasePool<T>(_ execute: () throws -> T) rethrows -> T {
```

Comment:
Rather than make this `internal` and `@inlinable`, is there any reason not simply to move it to the file with its new usage?

Reply:
@Lukasa I can revert it but I didn't see a reason not to make it available in tests etc.

Reply:
@Lukasa also it now needs to be usableFromInline/inlinable which requires internal, I'll move it to the SelectableEventLoop file though